src/main/java/org/assertj/vavr/api/AbstractMapAssert.java

```diff
@@ -192,6 +190,25 @@ public SELF containsOnly(Iterable<Tuple2<KEY, VALUE>> entries) {
         return myself;
     }
 
+    /**
+     * Verifies that the actual map contains only the given entries and nothing else, <b>in order</b>.<br>
+     * This assertion should only be used with maps that have a consistent iteration order (i.e. don't use it with
+     * {@link io.vavr.collection.HashMap}, prefer {@link #containsOnly} methods in that case).
+     *
+     * @param entries the given entries.
+     * @return {@code this} assertions object
+     * @throws NullPointerException     if the given entries array is {@code null}.
+     * @throws AssertionError           if the actual map is {@code null}.
+     * @throws IllegalArgumentException if the given entries array is empty.
+     * @throws AssertionError           if the actual map does not contain the given entries with same order, i.e. the actual map
+     *                                  contains some or none of the given entries, or the actual map contains more entries than the given ones
+     *                                  or entries are the same but the order is not.
+     */
```

Comment:
The goal of this assertion is to check entries (in order) only - without checking values? Am I right?

Reply:
But to check whether an entry is in the order of the actual map we have to check values as well, right? Checking keys only is not sufficient, I guess.